app/assets/stylesheets/application.scss

```diff
@@ -72,6 +72,24 @@ body {
   background-color: #fff;
 }
 
+.game-search {
+  width: 100%;
+  height: 100%;
+  background: #EE9D1C;
+}
+
+.btn-game-search {
+  color: rgb(255, 255, 255)!important;
+  background-color: black!important;
+  border-color: black!important;
+}
+
+.btn-game-search:hover {
+  color: #000000!important;
+  background-color: #ffffff!important;
+  border-color: #ffffff!important;
+}
+
```

Comment:
Created some new divs and classes for the game search section on the welcome page

Reply:
This looks great. I'm excited to merge to see what it looks like